forge/test/mlir/test_loss.py

```diff
@@ -78,20 +76,24 @@ def test_cross_entropy_loss(prediction_shape):
         (128, 128),
     ],
 )
-@pytest.mark.parametrize("reduction", ["mean", "sum"])
-def test_mse_loss(prediction_shape, reduction):
```

Comment:
Why is the MSE test removed?

Reply:
Accidentally, thanks for noticing, returning it ASAP!